client/rest-high-level/src/main/java/org/elasticsearch/client/core/TermVectorsRequest.java

```diff
@@ -197,6 +220,9 @@ public boolean getRealtime() {
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
+        builder.field("_index", index);
+        builder.field("_type", type);
```

Comment:
I know @jtibshirani has done some work on removing types from 7.0, and I dont know how this may or may not be impacted. @jtibshirani does this need to be different at all based on the work you did in #35421 ?